tensorflow_privacy/privacy/membership_inference_attack/data_structures.py

```diff
@@ -529,6 +529,39 @@ def __str__(self):
     ])
 
 
+@dataclass
+class SingleRiskScoreResult:
+  """Results from computing privacy risk scores.
+  this part is quite preliminary: it shows how to leverage privacy risk score to perform attacks with thresholding on risk score
```

Comment:
What does it mean for this part to be preliminary? Is it going to be updated in the future?

Reply:
By saying preliminary, I mean that we may leverage privacy risk scores for more analysis in the future. 
Currently, I follow our paper to set a threshold on the privacy risk score and compute precision and recall values.
With full training and test privacy risk scores, we can also return sample indices that have high privacy risk scores or implement other functionalities based on what we want.
I change the comment a bit in the updated code.